components/Buttons/tests/unit/ButtonsThemingTest.swift

```diff
@@ -216,5 +216,134 @@ class ButtonsThemingTest: XCTestCase {
     XCTAssertEqual(button.minimumSize.height, 36.0, accuracy: 0.001)
   }
 
+  func testFloatingButtonSecondaryThemeWithNoCustomThemes() {
+    // Given
+    let button = MDCFloatingButton()
+    let defaultButton = MDCFloatingButton()
+    let scheme = MDCContainerScheme()
+    let colorScheme = MDCSemanticColorScheme(defaults: .material201804)
+    let typographyScheme = MDCTypographyScheme(defaults: .material201804)
+
+    // When
+    button.applySecondaryTheme(withScheme: scheme)
+
+    // Then
+    XCTAssertEqual(button.backgroundColor(for: .normal), colorScheme.secondaryColor)
+    XCTAssertEqual(button.imageTintColor(for: .normal), colorScheme.onSecondaryColor)
+    // XCTAssertEqual(button.titleColor(for: .normal), colorScheme.onSecondaryColor)
```

Comment:
Please add a TODO for this line so we can track the work to fix it.

Reply:
Yes. I am going to mention your fix in the TODO to make it explicit.

Reply:
Addressed in 1b3abf543a18d093735c7636cbe962a3831b11dd